### ISSUE TITLE
IE9 Throws Script Error with AddThis

### ISSUE THREAD

**Issue by justrhysism** [LeaVerou/prefixfree#35]
When AddThis is included on a page, IE9 throws a script error. Doesn't appear to effect anything visually.

**Error thrown:**
 SCRIPT5: Access is denied. 
 prefixfree.min.js, line 2 character 336

**AddThis stylesheet URL:**
http://s7.addthis.com/static/r07/widget74.css

**Comment by MulletBoy** [comment]
Prefixing cross-origin linked stylesheets is not supported

**Comment by justrhysism** [comment]
Yes I know, that's why I'm surprised as to _why_ it's throwing the error. The script shouldn't be trying to access the other stylesheets (nor do I want it to).

**Comment by joelpittet** [comment]
is it possible to exclude AddThis from it's parsing?

**Comment by justrhysism** [comment]
Would happily do so, I'm just unsure _how_.

AddThis (or anything which is loaded externally, really) doesn't need to be parsed at all.

**Comment by joelpittet** [comment]
Looks like a work around may be to add a data-noprefix  attribute to your link. I am going to see if I can't give this a try.

**Comment by joelpittet** [comment]
For a more permanent solution, maybe using something like this parse URI script
http://blog.stevenlevithan.com/archives/parseuri

To double check for the Same Origin Policy

The parser could be simpler in this case but not too simple that it doesn't get those edge cases like //domain.com  or https://domain.com   or the http://user:pass@domain.com

**Comment by joelpittet** [comment]
This would be a lighter and possibly simpler approach:

```
var url = 'http://www.google.com';
var loc = window.location, a = document.createElement('a');
a.href = url;
var isSameDomain = a.hostname === loc.hostname && a.port === loc.port && a.protocol === loc.protocol;
```

from:
http://stackoverflow.com/questions/9404793/check-if-same-origin-policy-applies

Example:
http://jsfiddle.net/yJELu/
